tst/EnergyPlus/unit/PlantHeatExchangerFluidToFluid.unit.cc

```diff
@@ -2446,22 +2446,22 @@ TEST_F(EnergyPlusFixture, PlantHXControl_CoolingSetpointOnOffWithComponentOverri
     DataPlant::PlantLoop(1).FluidName = "WATER";
     DataPlant::PlantLoop(1).LoopSide(2).Branch(1).Comp(1).Name = PlantHeatExchangerFluidToFluid::FluidHX(1).Name;
     DataPlant::PlantLoop(1).LoopSide(2).Branch(1).Comp(1).TypeOf_Num = DataPlant::TypeOf_FluidToFluidPlantHtExchg;
-    DataPlant::PlantLoop(1).LoopSide(2).Branch(1).Comp(1).NodeNumIn = PlantHeatExchangerFluidToFluid::FluidHX(1).SupplySideLoop.InletNodeNum;
+    DataPlant::PlantLoop(1).LoopSide(2).Branch(1).Comp(1).NodeNumIn = PlantHeatExchangerFluidToFluid::FluidHX(1).SupplySideLoop.inletNodeNum;
 
     DataPlant::PlantLoop(2).Name = "HX demand side loop ";
     DataPlant::PlantLoop(2).FluidIndex = 1;
     DataPlant::PlantLoop(2).FluidName = "WATER";
     DataPlant::PlantLoop(2).LoopSide(1).Branch(1).Comp(1).Name = PlantHeatExchangerFluidToFluid::FluidHX(1).Name;
     DataPlant::PlantLoop(2).LoopSide(1).Branch(1).Comp(1).TypeOf_Num = DataPlant::TypeOf_FluidToFluidPlantHtExchg;
-    DataPlant::PlantLoop(2).LoopSide(1).Branch(1).Comp(1).NodeNumIn = PlantHeatExchangerFluidToFluid::FluidHX(1).DemandSideLoop.InletNodeNum;
+    DataPlant::PlantLoop(2).LoopSide(1).Branch(1).Comp(1).NodeNumIn = PlantHeatExchangerFluidToFluid::FluidHX(1).DemandSideLoop.inletNodeNum;
 
     PlantHeatExchangerFluidToFluid::FluidHX(1).DemandSideLoop.MassFlowRateMax = 2.0;
     PlantHeatExchangerFluidToFluid::FluidHX(1).ControlSignalTemp = PlantHeatExchangerFluidToFluid::DryBulbTemperature;
-    PlantHeatExchangerFluidToFluid::FluidHX(1).OtherCompSupplySideLoop.InletNodeNum = 5;
-    PlantHeatExchangerFluidToFluid::FluidHX(1).OtherCompSupplySideLoop.LoopNum = 1;
-    PlantHeatExchangerFluidToFluid::FluidHX(1).OtherCompSupplySideLoop.LoopSideNum = 2;
-    PlantHeatExchangerFluidToFluid::FluidHX(1).OtherCompSupplySideLoop.BranchNum = 2;
-    PlantHeatExchangerFluidToFluid::FluidHX(1).OtherCompSupplySideLoop.CompNum = 1;
+    PlantHeatExchangerFluidToFluid::FluidHX(1).OtherCompSupplySideLoop.inletNodeNum = 5;
```

Comment:
When there's a whole slew of references to an object like this, feel free to make a reference to it `auto & thisFluidHX = Blah::FluidHX(1);`.  Obviously not worth a whole new commit, but just feel free while you're cleaning up the unit tests like you did here.  :+1: